### DISCUSSION TITLE
Add Arabic Language

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [andrewdwallo/filament-companies#59] [Feature requests]
Please add arabic language file

[default.txt](https://github.com/andrewdwallo/filament-companies/files/11140603/default.txt)

**Comment by andrewdwallo** [comment]
You don't want to submit a pull request?

**Reply by mstfkhazaal** [reply]
No

**Reply by mstfkhazaal** [reply]
And please add  command in cmd to override language

**Reply by mstfkhazaal** [reply]
Fix rtl like this 
![image](https://user-images.githubusercontent.com/47450090/229576958-1b0fecbb-47bc-48bb-9bc1-afd5b5512fce.png)

**Reply by andrewdwallo** [reply]
You are welcome to submit pull request. I am not going to do every request you have. If you want it then submit a pull request.
